doc/installation.rst

```diff
@@ -1,65 +1,50 @@
 ..
-  SPDX-FileCopyrightText: 2016 - 2023 The Atlite Authors
+  SPDX-FileCopyrightText: Contributors to Atlite <https://github.com/pypsa/atlite>
 
   SPDX-License-Identifier: CC-BY-4.0
 
-############
+##############
 Installation
-############
+##############
 
-There are three possibilities to install atlite:
 
-* conda::
+Getting Python
+==============
 
-    conda install -c conda-forge atlite
+If it is your first time with Python, we recommend `conda
+<https://docs.conda.io/en/latest/miniconda.html>`_, `mamba
+<https://github.com/mamba-org/mamba>`_ or `pip
```

Comment:
```suggestion
<https://docs.conda.io/en/latest/miniconda.html>`_ or `pip
```

Let's get rid of `mamba`. The solver has been incorporated as default into `conda` already and we don't want to confuse the first time Python project users.